WordPress/Classes/ViewRelated/Post/PublishSettingsViewController.swift

```diff
@@ -0,0 +1,179 @@
+import Foundation
+import CocoaLumberjack
+import WordPressShared
+
+private enum PublishSettingsCell: CaseIterable {
+    case dateTime
+
+    var title: String {
```

Comment:
Is this used? I commented it out with no ill effect. Maybe it's not needed anymore?

Reply:
This isn't used currently but I was planning to use it later on when their are multiple cell states. I've removed if for now and will re-add in a future PR if needed.

Reply:
That'd be great! Thanks!